tests/main_test.go

```diff
@@ -89,7 +89,7 @@ func TestMain(m *testing.M) {
 		tConfigs[i].Sync.Firewall.Enabled = false
 		tConfigs[i].Network.EnableMdns = true
 		tConfigs[i].Network.NetworkKey = util.TempFilePath()
-		tConfigs[i].Network.Listens = []string{"/ip4/127.0.0.1/tcp/0"}
+		tConfigs[i].Network.Listens = []string{"/ip4/127.0.0.1/tcp/0", "/ip4/127.0.0.1/udp/0/quic"}
```

Comment:
I think if we use 127.0.0.1 for tests it perfectly works on shell pipeline runners but in docker pipeline runners it would be better to use 0.0.0.0 which work's like 127.0.0.1 in shell runners.

Reply:
Same as above ☝️